recipes/inline-model-metrics/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "inline-model-metrics" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/inline_model_metrics-{{ version }}.tar.gz
+  sha256: 0f0ffef1ae2d5450698f926e79124a49de2edc540a62122d6b59754d05fa5fb8
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: True  # [win or osx]
```

This looks like it's pure Python:
```suggestion
  noarch: python
```

we can ignore the resulting CI failures from `metoffice-afterburner` only being available on Linux (noarch packages are only normally built on Linux).